x-pack/legacy/plugins/siem/public/components/header_page_new/index.tsx

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
This component is to match the new design of the header like in the new design of case.
![image](https://user-images.githubusercontent.com/189600/74265320-5d460580-4cd0-11ea-94dd-3aa62a7b1ff5.png)

Reply:
yes, may end up combining into one header component. i left all the same functionality as the original